aspnetcore/fundamentals/servers/kestrel/samples/6.x/Http3Sample/Program.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Server.Kestrel.Https;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Http3Sample
+{
+    public class Program
+    {
+        #region snippet_UseHttp3
+        public static void Main(string[] args)
+        {
+            var cert = CertificateLoader.LoadFromStoreCert("localhost", StoreName.My.ToString(), StoreLocation.CurrentUser, false);
```

Comment:
Remove. Use default ASP.NET Core dev cert

Reply:
Example of this?

Reply:
You've already done it with `UseHttps()`, this CertificateLoader line is now redundant and can be removed.

Reply:
```suggestion
```